### ISSUE TITLE
horribly confused by fneg

### ISSUE THREAD

**Issue by magcius** [aldelaro5/ghidra-gekko-broadway-lang#3]
has been an issue for a long time,

but ghidra often emits fneg as `(ulonglong)(double)fVar1 & 0x7FFFFFFFFFFFFFFF | ~(ulonglong)(double)fVar1 & 0x800000000000)` which is just annoying

![image](https://user-images.githubusercontent.com/40890/71773583-3e579500-2f14-11ea-97e6-3e1fa363738a.png)

**Comment by aldelaro5** [comment]
oh I found the issue and I need to do some explaining because this SHOULD have been fixed long ago and I just...forgot.

Basically, this was a ghidra issue that I submitted a PR to fix and it got merged https://github.com/NationalSecurityAgency/ghidra/pull/387

The problem is the sleigh I made for gekko works this way: I copy pasted the ppc of ghidra and added the instructions on top as well as the ps register. It was not possible for me to JUST add my instructions because I had to redefine some registers stuff as well as some opcode parsing bits that basically forced me to copy the files to redefine them. It's been a while I made this so maybe this has changed, but until I actively come back to work on gc stuff (Q1 2020), I'll leave it like this for the time being.

The problem is although the issue was fixed in ghidra, I never updated my own sleigh to reflect it. Since the fix is trivial, I will push the fix later today.
